recipe/meta.yaml

```diff
@@ -53,6 +60,9 @@ requirements:
     - liblapack
   run:
     - python
+{% if dev != '' %}
+    - _numpy_rc
+{% endif %}
```

Comment:
Ah sorry misunderstood this detail ( https://github.com/conda-forge/numpy-feedstock/issues/311#issuecomment-2088915857 ). Now understand the `main` labeled packages are unavailable

Have moved them back

Please double check though

<img width="1840" alt="Screenshot 2024-05-01 at 9 22 26 PM" src="https://github.com/conda-forge/numpy-feedstock/assets/3019665/589a70ba-7052-4591-8b80-d65bcdd1e598">